packages/protocol/CHEATSHEET.md

```diff
@@ -0,0 +1,47 @@
+| `migrate`                               | `devchain:migrate`                            |
+| `devchain:reset`                        | `devchain:reset`                              |
+| `check-opcodes`                         | `release:check-opcodes`                       |
+| `check-versions`                        | `release:check-versions`                      |
+| `determine-release-version`             | `release:determine-release-version`           |
+| `make-release`                          | `release:make`                                |
+| `verify-deployed`                       | `release:verify-deployed`                     |
+| `verify-release`                        | `release:verify-release`                      |
+| `pull-submodules`                       | `submodules:pull`                             |
+| `delete-submodules`                     | `submodules:delete`                           |
+| `compare-git-tags`                      | `tags:compare`                                |
+| `view-tags`                             | `tags:view`                                   |
+| `...`                                   | `test`                                        |
+| `test:coverage`                         | `test:coverage`                               |
+| `gas`                                   | `test:gas`                                    |
+| `quicktest`                             | `test:quicktest`                              |
+| `test:release-snapshots`                | `test:release-snapshots`                      |
+| `test:scripts`                          | `test:scripts`                                |
+| `test`                                  | `test:truffle`                                |
+| `console`                               | `truffle:console`                             |
+| `govern`                                | `truffle:govern`                              |
+| `migrate`                               | `truffle:migrate`                             |
+| `set_block_gas_limit`                   | `truffle:set-block-gas-limit`                 |
+| `truffle-verify`                        | `truffle:verify`                              |
+| `prepare_contracts_and_abis_publishing` | `utils:prepare-contracts-and-abis-publishing` |
+| `prepare_devchain_anvil_publishing`     | `utils:prepare-devchain-anvil-publishing`     |
+| `sourcify-publish`                      | `utils:sourcify-publish`                      |
+| `validate_abis_exports`                 | `utils:validate-abis-exports`                 |
+| `download-artifacts`                    | `...`                                         |
+| `generate-stabletoken-files`            | `...`                                         |
+| `revoke`                                | `...`                                         |
+| `upload-artifacts`                      | `...`                                         |
```

Comment:
I think these we can just ignore, they are very old

Reply:
Yeah, it's just for reference that they were deleted 😉